modules/bpi/bpi.images.inc

```diff
@@ -88,32 +91,45 @@ function bpi_assets_form($form, $form_state, $assets = array()) {
  * @ingroup forms
  */
 function bpi_assets_form_submit($form, &$form_state) {
-  $input = array_keys($form_state['input']);
+  $bpi_assets = $_SESSION['bpi']['assets'];
+
```

Comment:
We need session values to send values from the main content form and the assets popup. The original code uses this as well (cf. https://github.com/ding2/ding2/blob/master/modules/bpi/bpi.images.inc#L97-L99), but I moved it to the top of the function for clarity.